worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Controller.scala

```diff
@@ -332,6 +332,12 @@ private[deploy] class Controller(
                 }
 
                 val fileWriter = location.asInstanceOf[WorkingPartition].getFileWriter
+                val fileInfo =
+                  if (null != fileWriter.getDiskFileInfo) {
```

Comment:
You could update `PartitionDataWriter#getFilePath` and use this method.

Reply:
I have updated code.